action/protocol/poll/staking_command.go

```diff
@@ -109,7 +109,7 @@ func (sc *stakingCommand) CalculateUnproductiveDelegates(
 	sr protocol.StateReader,
 ) ([]string, error) {
 	if sc.useV2(ctx, sr) {
-		sc.stakingV2.CalculateUnproductiveDelegates(ctx, sr)
+		return sc.stakingV2.CalculateUnproductiveDelegates(ctx, sr)
```

Comment:
this might be a bug, if confirmed need to fix in a hard-fork, revert this

Reply:
ok